.changeset/few-snails-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-backend/express': minor
+---
+
+update `getConfiguredAudience` with explicit return and not use `toString()`
```

Comment:
```suggestion
---
'@commercetools-backend/express': patch
---

Fix issue with invalid `audience` in case the request contains query parameters.
```